spec.html

```diff
@@ -6250,14 +6250,15 @@ <h1>CreateIntrinsics ( _realmRec_ )</h1>
         1. Set _realmRec_.[[Intrinsics]] to _intrinsics_.
         1. Let _objProto_ be ObjectCreate(*null*).
         1. Set _intrinsics_.[[%ObjectPrototype%]] to _objProto_.
-        1. Let _throwerSteps_ be the algorithm steps specified in <emu-xref href="#sec-%throwtypeerror%"></emu-xref> for the %ThrowTypeError% function.
-        1. Let _thrower_ be ! CreateBuiltinFunction(_throwerSteps_, &laquo; &raquo;, _realmRec_, *null*).
-        1. Set _intrinsics_.[[%ThrowTypeError%]] to _thrower_.
         1. Let _noSteps_ be an empty sequence of algorithm steps.
         1. Let _funcProto_ be ! CreateBuiltinFunction(_noSteps_, &laquo; &raquo;, _realmRec_, _objProto_).
         1. Set _intrinsics_.[[%FunctionPrototype%]] to _funcProto_.
-        1. Call _thrower_.[[SetPrototypeOf]](_funcProto_).
-        1. Perform AddRestrictedFunctionProperties(_funcProto_, _realmRec_).
+        1. Let _throwerSteps_ be the algorithm steps specified in <emu-xref href="#sec-%throwtypeerror%"></emu-xref> for the %ThrowTypeError% function.
+        1. Let _thrower_ be ! CreateBuiltinFunction(_throwerSteps_, &laquo; &raquo;, _realmRec_, _funcProto_).
+        1. Set _thrower_.[[Extensible]] to *false*.
```

Comment:
should this use SetIntegrityLevel instead?

Reply:
I went for this approach because the definition says the extensible slot is false, not that the function is frozen or sealed.